src/components/select/_select.scss

```diff
@@ -3,6 +3,15 @@
 
 @include exports("select") {
   .govuk-c-select {
+    @include govuk-responsive-margin($govuk-spacing-responsive-6, "bottom");
+  }
+
+  .govuk-c-select--error {
+    padding-left: $govuk-spacing-scale-3;
```

Comment:
If we decide to keep error states in here (as opposed to only in the form-group) we might want to think about abstracting this into a mixin or something.